.github/workflows/ci.yaml

```diff
@@ -82,12 +82,12 @@ jobs:
         # When changing versions here, check that the version exists at:
         # https://github.com/yannh/kubernetes-json-schema
         k8s:
+          - v1.28.2
+          - v1.27.6
```

Comment:
v1.15-v1.18 is deprecated https://github.com/yannh/kubernetes-json-schema/commit/9c0fe55fdb6c660a4b8115220db419a87c7068f1